ios/PolyPodApp/PodApi/PolyIn/Quads/PolyIn+Add.swift

```diff
@@ -2,9 +2,9 @@ import UIKit
 import CoreData
 
 extension PolyIn {
-    func addQuads(quads: [ExtendedData], completionHandler: (Bool) -> Void) {
+    func addQuads(quads: [ExtendedData], completionHandler: (Error?) -> Void) {
         guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else {
-            completionHandler(false)
+            completionHandler(PodApiError.noAppDelegate)
```

Comment:
While some `PolyIn` implementations where explicitly returning an error, others were just returning a boolean indicating success - normalised this. With the exception of `polyIn.has`, see below.

Reply:
You mean hasQuads. OK, that explains what you do above.